cli/flox/doc/flox-activate.md

```diff
@@ -90,11 +92,21 @@ hook was defined.
     but they will not be pushed,
     and they will not persist when using a remote environment with `-r`.
 
+`$FLOX_ENV_PROJECT`
```

Comment:
Suggestion: I'd mention that this variable is _provided_ to the user i.e. they don't need to set it, and probably shouldn't set it.

Reply:
Fixed